xml/System.Security.Cryptography.X509Certificates/X509CertificateCollection.xml

```diff
@@ -1141,8 +1141,8 @@
         <Parameter Name="value" Type="System.Object" Index="0" FrameworkAlternate="netcore-1.0;netcore-1.1;netstandard-1.3;netstandard-1.4;netstandard-1.6" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="value">The object to remove from the collection.</param>
+        <summary>Removes the first occurrence of a specific object from the collection.</summary>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
         <remarks>
           <format type="text/markdown"><![CDATA[

## Remarks

This member is an explicit interface member implementation. It can be used only when the <xref:System.Security.Cryptography.X509Certificates.X509CertificateCollection> instance is cast to an <xref:System.Collections.IList> interface.  

           ]]></format>
         </remarks>
```